### ISSUE TITLE
Because it takes so much to compile it ?

### ISSUE THREAD

**Issue by raymand211092** [martinetd/matrirc#11]
![imagen](https://github.com/martinetd/matrirc/assets/50679989/aa193394-3bc6-4132-adf9-528eae7cb65d)

**Comment by martinetd** [comment]
I'm afraid that's just how rust and the matrix rust sdk makes it to be, matrirc itself doesn't do much.
If you build in release mode it should be much smaller (38MB here, which I still consider huge, but it's out of my reach)

If you find a way to make it smaller I'll be glad to use it

**Comment by aiobofh** [comment]
Yeah, Rust projects are beasts.... That's the price you have to pay to build monolithic distributions. With the (potential) upside of having less integration work to fiddle with against various libs in various distros.

I wish there was a mature, easy to use, C library/SDK for Matrix :violin:
